src/ops/stash.rs

```diff
@@ -0,0 +1,181 @@
+use super::{Action, OpTrait};
+use crate::{items::TargetData, prompt::PromptData, state::State, term::Term, Res};
+use derive_more::Display;
+use std::{process::Command, rc::Rc};
+use tui_prompts::State as _;
+
+#[derive(Default, Clone, Copy, PartialEq, Eq, Debug, Display)]
+#[display(fmt = "Stash")]
+pub(crate) struct Stash;
+impl OpTrait for Stash {
+    fn get_action(&self, _target: Option<&TargetData>) -> Option<Action> {
+        Some(stash_push_action(["--include-untracked"]))
```

Comment:
We don't have arguments yet, and this differs from git/magit (which is fine), but it might come as a surprise to some.
I'm thinking the `#[display(fmt = "Stash")]` could include this information for now. `"Stash (include untracked)`.
What do you think?

Reply:
This is very reasonable

Reply:
<img width="470" alt="Screenshot 2567-03-18 at 14 00 02" src="https://github.com/altsem/gitu/assets/83108115/49efc7cc-0fc1-4b50-b577-d37f1165f64b">
I added this message only for 'z' stash because for other ones there is no enough space